components/monitoring/prometheus/base/monitoringstack/monitoringstack.yaml

```diff
@@ -73,7 +96,6 @@ spec:
         - '{__name__="kube_pod_status_unschedulable", namespace!~".*-tenant|openshift-.*|kube-.*"}'
         - '{__name__="kube_pod_container_status_waiting_reason", namespace!~".*-tenant|openshift-.*|kube-.*"}'
         - '{__name__="kube_pod_status_phase", namespace!~".*-tenant|openshift-.*|kube-.*"}'
-        - '{__name__="kube_pod_status_unschedulable", namespace!~".*-tenant|openshift-.*|kube-.*"}'
```

Comment:
Why did you remove this one?

Reply:
It's was accidentally copied twice originally. You can see it's also on line 96.

Reply:
I see it, thanks :)